.circleci/config.yml

```diff
@@ -45,7 +45,7 @@ jobs:
           command: yarn lint
       - run:
           name: Test JavaScript
-          command: yarn test
+          command: yarn test --runInBand
```

Comment:
This is the cause of slow tests, as they are run serially. Try to leverage number of cores you can use on your CI, try different ones, e.g. `yarn test -w=4` for 4 parallel workers. Some CIs are better and some worse in using multiple cores, that's why you need to play with it a bit

Reply:
https://circleci.com/gh/Semantic-Org/Semantic-UI-React/5711

Removal of this option causes build failure. Also, stardust-ui/react#53

Reply:
That's why I was talking about using certain number of workers, not all available (depending on CI, it can tell Jest it has even 128 cpus). You should be good to run jest without flags locally, as you probably have up to 8 cores and enough memory.